app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,21 @@
+class CommentsController < ApplicationController
+  def new
+    @comment = Comment.new
+  end
+
+  def create
+    @comment = Comment.new(comment_params)
+    if @comment.save
+      redirect_to user_posts_path(current_user)
+    else
+      render :new
+    end
+  end
```

Comment:
- [ ] When I try to make a comment, I get the following error, kindly fix this issue and make sure adding comments works properly.
>![image](https://user-images.githubusercontent.com/53402600/216767377-49821490-58f6-448b-a5cb-2282705ef557.png)